examples/event_handling/resample.py

```diff
@@ -1,9 +1,15 @@
 """
-========
-Resample
-========
+===============
+Resampling Data
+===============
+
+
+Downsampling lowers the sample rate or sample size of a signal. In
+this tutorial, the signal is downsampled when the plot is adjusted
+through dragging and zooming.
```

Comment:
It does what it claims, but in a weird way.  It always down-samples to every 5th point but also clips the data.